portfolio/src/main/java/com/google/sps/authentication/servlets/LoginServlet.java

```diff
@@ -13,8 +14,24 @@
 @WebServlet("/login")
 public final class LoginServlet extends HttpServlet {
 
+    private static final String GUEST_USER_NAME = "stranger";
```

Comment:
Why not just "guest" haha, "stranger" sounds so cold LOL

Reply:
ok fixed haha